build.gradle

```diff
@@ -106,5 +106,6 @@ buildScan {
 
 ext {
     daggerVersion = '2.22.1'
-    fluxCVersion = '1.5.1'
+    // FIXME update to FluxC release
+    fluxCVersion = 'd058757009410e19dfbf86cb0a6895d90b97cf43'
```

Comment:
you can update this since it's merged now 👍

Reply:
I just wanted to wait until this PR got some testing before tagging a fluxc release in case any issues were found that needed additional fluxc changes. But hopefully you're right, and I'll just be tagging this commit. 😄